src/api.js

```diff
@@ -117,6 +117,24 @@ function qmlweb_parse($TEXT, document_type, exigent_mode) {
     token_error(S.token, "Unexpected token " + S.token.type + " " + S.token.val + ", expected " + type + " " + val);
   };
 
+  function qml_get_attached_object(type, val) {
+      if (type === "name" && qml_is_element(val)) {
+          return "$QmlWebGetAttachedObject('" + val + "')";
```

Comment:
I don't think this should be on the parser side. Parser should just provide a consumable AST.